woocommerce/woocommerce
# Install and set up BuildKite test analytics

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This adds the Buildkite Test Analytics reporter to our e2e PR runs to collect stats for PR run time, failure rates, flaky test rates etc...  The results can be seen here: https://buildkite.com/organizations/woocommerce-1/analytics/suites/woocommerce-core-e2e?branch=all+branches



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. E2E tests pass in CI.
2. If you re-run the E2E tests, a new entry should appear on https://buildkite.com/organizations/woocommerce-1/analytics/suites/woocommerce-core-e2e?branch=all+branches for that run.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
